pkgs/top-level/all-packages.nix

```diff
@@ -6576,18 +6576,16 @@ with pkgs;
 
   mlton = callPackage ../development/compilers/mlton { };
 
-  mono = mono40;
+  mono = mono4;
+
+  mono4 = mono40;
```

Comment:
How about `mono = mono48`? I am pretty sure there have been several fixed security issues in the 4.x series.

Reply:
Sure, via  mono -> mono4 -> mono48 (in stable), and mono5 in master. Just idk what apps can break.  I'd happy to leave  only 4.8 and latest 5.x (but at least 2 apps require exact 4.6